website/docs/guides/text-search.mdx

```diff
@@ -0,0 +1,97 @@
+It is widely used in applications like search engines, e-commerce platforms, documentation searches, and content management systems.
+
+## Understanding text indexes
+
+A full-text index is fundamentally different from a regular database index.
+Instead of simply mapping fields to values, it:
+
+- Tokenizes text (splits them into words or phrases).
+- Removes stop words (such as "is", "the", "and").
+- Applies stemming (so "running" and "run" are treated as the same).
+- Assigns weights based on frequency, importance, or custom ranking logic.
+
+FerretDB supports full-text search capabilities.
+
+A full text search index creation takes the following parameters:
+
+| Field            | Description                                                                                |
+| ---------------- | ------------------------------------------------------------------------------------------ |
+| name             | A custom name for the index, useful for reference.                                         |
+| weights          | Assigns weighting to fields (higher values mean more relevance in search). Default is `1`. |
+| default_language | Specifies the language used for stemming (default: "english").                             |
+| caseSensitive    | Enables case-sensitive search.                                                             |
+
+:::note
+FerretDB only supports one text index per collection.
+:::
+
+## Single full text index
+
+Single full text index is created on a single field in a collection.
+
+### Creating a text index
+
+To create a text index, use the `createIndex` command with the field you want to index and the type set to `'text'`.
+
+<!-- markdownlint-disable MD033 -->
+
+<CodeBlock language="js">{CreateIndex}</CodeBlock>
+
+This command creates a full text index on the `summary` field in a `books` collection.
+
+Insert the following documents into the `books` collection:
+
+<CodeBlock language="js">{MyDataSet}</CodeBlock>
+
+### Performing a full text search
+
+Let's run a basic full text search query to find all documents that contain the word "drama" in the `summary` field.
+
+<CodeBlock language="js">{SearchQuery}</CodeBlock>
+
+This query returns all documents where the `summary` field contains the word "drama".
+
+<CodeBlock language="js">{TextSearchResponse}</CodeBlock>
+
+## Compound text index
+
+Compound text index creates an index on multiple fields.
+Ensure to drop the existing index before creating a new one.
+
+Let's create a compound text index on the `title` and `summary` fields.
+
+<CodeBlock language="js">{CompoundIndex}</CodeBlock>
+
+## Relevance score
+
+When you perform a full-text search, a relevance score is assigned to each document based on how well it matches the search query.
+The score is calculated based on factors like word frequency, proximity, and custom weights.
+Relevance scores are calculated using the assigned weights and the frequency of the search terms in the document.
```

Comment:
Above two sentences are very similar, do we need both sentences.

Reply:
Yeah, looks like an editing error